actionx/ACTIONX_WCONINJE.DATA

```diff
@@ -322,6 +322,8 @@ RPTSCHED
 --
 --       RESTART CONTROL BASIC = 4 (ALL=2, YEARLY=4, MONTHLY=5, TSTEP=6)
 --
+RPTRST
+         BASIC=5                                                               /
```

Comment:
Note that this particular change reverts PR #635.  Is that intentional?

Reply:
Yes.

Reply:
>> Note that this particular change reverts PR #635. Is that intentional?
>
> Yes.

In that case I'll have @joakim-hove do the review here.  I don't think the underlying issue that prompted #635 has been fixed in the interim.

Reply:
> > > Note that this particular change reverts PR #635. Is that intentional?
> > 
> > 
> > Yes.
> 
> In that case I'll have @joakim-hove do the review here. I don't think the underlying issue that prompted #635 has been fixed in the interim.

Yes, and the change also breaks the test question. Highly appreciated if @joakim-hove takes over the review.

Reply:
Well; for the `ACTIONX_WCONINJE` I can only confirm that the `RPTRST BASIC=5` issue has not been fixed - getting the `RPTRST / RPTSCHED` functionality work has been *extremely* difficult :disappointed: 

Now I agree that the `BASIC=5` should eventually be fixed - but is there a real value in enforcing that for this particular test? I would encourage you to just revert this particular change.

Reply:
Done